.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,35 +1,23 @@
-# Introduction
-Awesome that you are interested in hosting an event!
-
-This is a template that we use here at Stockholm React JS.
-You can add / remove text from it but please try to keep the structure of it intact.
-
-# Your event intro
-The intro text of this event, use this to quickly explain what the event
-will be all about, be clear and try to get peoples attention. Don't write too much.
-
-# Hosted by
-**Company Name:**
-
-**Address:**
+# Description
+Describe your event here for the community so that they can better help you in realizing the event.
 
```

Comment:
I don't think it's necessary for a first version, but maybe we could add a checklist here, something along the lines of:

- [ ] I have read the [hosting guide](https://www.example.com)
- [ ] As a host we agree to follow the [code of conduct](https://github.com/stockholm-react-js/governance/blob/master/CODE_OF_CONDUCT.md)
- [ ] If serving food, a vegan alternative will be present
- [ ] If serving drinks, non-alcohol alternatives will be present

Just examples, should just contain the most important stuff we want to remind about I guess. :)